internal/clients/http/azure/create_vm.go

```diff
@@ -20,13 +20,14 @@ import (
 	"github.com/RHEnVision/provisioning-backend/internal/clients"
 	"github.com/RHEnVision/provisioning-backend/internal/models"
 	"github.com/RHEnVision/provisioning-backend/internal/ptr"
+	"github.com/RHEnVision/provisioning-backend/internal/telemetry"
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/codes"
 )
 
 var subscriptionId string
 
-const TraceName = "github.com/RHEnVision/provisioning-backend/internal/clients/http/azure"
+const TraceName = telemetry.TracePrefix + "internal/clients/http/azure"
```

Comment:
This looks better, also we already had typos in there "EnVision" vs "RhEnVision".